R/dt_summary.R

```diff
@@ -170,21 +171,22 @@ dt_summary_build <- function(data,
     if (identical(groups, grand_summary_col)) {
 
       select_data_tbl <-
-        cbind(
-          stub_df[c("groupname", "rowname")],
-          data_tbl
-        )[, -2] %>%
+        data_tbl %>%
+        dplyr::select(!!columns) %>%
         dplyr::mutate(groupname = grand_summary_col) %>%
-        dplyr::select(groupname, columns)
+        dplyr::select(groupname, columns) %>%
```

Comment:
Probably `columns` needs to be `!!` or `!!!`

Reply:
fixed with `!!`